examples/runExample.ts

```diff
@@ -84,6 +85,10 @@ function outputToJson(record: Record<string, mariner.Issue[]>): void {
     fs.writeFileSync(config.outputFilePath, jsonResults);
 }
 
+contributorsFinder.fetchContributors(token, repositoryIdentifiers).then((contributors) => {
```

Comment:
Assuming the class is renamed to be a Finder, this function should probably be renamed to `findContributors`. That's my quick reaction, and I could change my mind over time.